test/e2e_node/summary_test.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/onsi/gomega/types"
 )
 
-const restartCount = 3
+const restartCount = 1
```

Comment:
restartCount is part of ContainerStatus: https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/types.go#L1909